### PR TITLE
Add podmin email to 500 error page

### LINKED ISSUE #4016: Add a link to the mail address of the podmin on the error page
My pod is actually down (error 500) and I'd like to contact the podmin to inform him, but without Diaspora, I can't. I think it could be a good idea to improve the error page to add a link to the mail address of the podmin.

<bountysource-plugin>
---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/114479-add-a-link-to-the-mail-address-of-the-podmin-on-the-error-page?utm_campaign=plugin&utm_content=tracker%2F522&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F522&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by svbergerem**
Fixes #4016.

500 error page

### CODE REVIEW

File: app/views/errors/error_500.haml [hunk 1]
```diff
@@ -7,6 +7,10 @@
   Internal server error.
   Our bad! Sorry about that. :(
 
+- if AppConfig.admins.podmin_email?
+  %p
+    Drop us an email to <a href="mailto:#{AppConfig.admins.podmin_email}">#{AppConfig.admins.podmin_email}</a>.
```
Comment (diaspora-code-review) [inline app/views/errors/error_500.haml]: `= "..."` should be rewritten as `...`

**Comment by Flaburgan** [comment]
Cool! I guess this can be merged in a minor release too :)

File: app/views/errors/error_500.haml (hunk 1 above)
Comment (svbergerem) [inline app/views/errors/error_500.haml]: I don't think that this would work. ;) Thus this PR should be ready to merge.

**Comment by denschub** [comment]
Merged as 0213bbc02813d47f661f9a955f479d4989397a01, thanks much!

**Pull request closed without merging**